conductr_cli/constants.py

```diff
@@ -17,7 +16,6 @@
                                        '{}/plugins'.format(DEFAULT_CLI_SETTINGS_DIR))
 DEFAULT_SANDBOX_IMAGE_DIR = os.path.abspath(os.getenv('CONDUCTR_SANDBOX_IMAGE_DIR',
                                                       '{}/images'.format(DEFAULT_CLI_SETTINGS_DIR)))
-DEFAULT_SANDBOX_INTERFACE = os.getenv('CONDUCTR_SANDBOX_INTERFACE', host.loopback_device_name())
 DEFAULT_SANDBOX_ADDR_RANGE = os.getenv('CONDUCTR_SANDBOX_ADDR_RANGE', '192.168.1.0/24')
```

Comment:
You cannot use 127.0.0.1/24 because Docker containers wouldn't be able to distinguish their bind address from their host address.